test/DynamoCoreTests/Nodes/ListTests.cs

```diff
@@ -665,6 +665,19 @@ public void SortBy_SimpleTest()
 		}
 
         [Test]
+        public void SortBy_PreviewValueIsCorrectForFunction()
+        {
+            string openPath = Path.Combine(TestDirectory, @"core\list\SortBy_SimpleTest.dyn");
+            RunModel(openPath);
+
+            var preview = GetPreviewValueInString("42ac0cec-442f-4e4a-b629-1260f6db3d86");
+            Assert.AreEqual("Function()", preview);
+            
+            var value = GetPreviewValue("42ac0cec-442f-4e4a-b629-1260f6db3d86");
+            Assert.Null(value);
+        }
```

Comment:
This test seems to pass even before this fix. Can you create one that fails before this fix?